chatwoot/chatwoot
# chore: Create conversation API should accept assignee & team

## Description

This change enables the create conversation API to accept assignee and team. So outbound conversations can be assigned to the person who create the message.

Fixes #2679

## Type of change


## How Has This Been Tested?

The below steps are tested using POSTMAN.

1. Create a new Inbox
2. Create a new Agent
2. Create a new Team.
3. Create a new Contact.
4. Create a new Conversation using the above contact, inbox, agent, and team.
5. Verify the agent and team details in the API response. 

## Checklist:
